content/runner/docker/installation/linux.md

```diff
@@ -47,17 +47,19 @@ The Docker runner is configured using environment variables. This article refere
 The below command creates a container and starts the Docker runner. _Remember to replace the environment variables below with your Drone server details._
 
 ```
-$ docker run -d \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  -e DRONE_RPC_PROTO=https \
-  -e DRONE_RPC_HOST=drone.company.com \
-  -e DRONE_RPC_SECRET=super-duper-secret \
-  -e DRONE_RUNNER_CAPACITY=2 \
-  -e DRONE_RUNNER_NAME=${HOSTNAME} \
-  -p 3000:3000 \
-  --restart always \
-  --name runner \
+{{< highlight handlebars "linenos=table" >}}
+$ docker run --detach \
+  --volume=/var/run/docker.sock:/var/run/docker.sock \
+  --env=DRONE_RPC_PROTO={{DRONE_RPC_PROTO}} \
+  --env=DRONE_RPC_HOST={{DRONE_RPC_HOST}} \
+  --env=DRONE_RPC_SECRET={{DRONE_RPC_SECRET}} \
+  --env=DRONE_RUNNER_CAPACITY={{DRONE_RUNNER_CAPACITY}} \
+  --env=DRONE_RUNNER_NAME=${{DRONE_RUNNER_NAME}} \
```

Comment:
The $ can be removed before the bracket here. Thanks @teezzan

Reply:
I'll fix that. Thanks